fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -118,6 +119,16 @@ public class TableProperty implements Writable, GsonPostProcessable {
      */
     private TStorageFormat storageFormat = TStorageFormat.DEFAULT;
 
+    /*
+     * the default storage volume of this table.
+     * DEFAULT: depends on FE's config 'run_mode'
+     * run_mode == "shared_nothing": "local"
+     * run_mode == "shared_data": "default"
+     *
+     * This property should be set when creating the table, and can not be changed.
+     */
+    private String storageVolume;
```

Comment:
storageVolume is in properties, why having a specific variable for it?

Reply:
![image](https://user-images.githubusercontent.com/10295131/220577385-8a55e9e8-359b-4cc2-9ed9-924d3a0ccdac.png)

Reply:
Move storageVolume into `StorageInfo` class?